main.nf

```diff
@@ -362,7 +369,7 @@ workflow analysis_reference_based {
         multiqc(multiqc_config, 
                 multiqc_sample_names.out,
                 fastp.out.json_report.collect(), 
-                sortmerna.out.log.collect(), 
+                sortmerna_log.collect().ifEmpty([]), 
```

Comment:
that's interesting! so when the channel after the collect is empty, `[]` is submitted as an input to the multiqc process?

Reply:
yes, indeed!